src/codeu/chat/util/PersistenceLog.java

```diff
@@ -0,0 +1,187 @@
+package codeu.chat.util;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.User;
+import codeu.chat.server.Controller;
+import codeu.chat.server.Server;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Utility class for writing and reading to/from the log
+ */
+public class PersistenceLog {
+
+  private static final String USER = "ADD-USER";
```

Comment:
make these public and use these in the controller to avoid having to add new transactions to both places.